### ISSUE TITLE
Empty Form Labels - Create New Generic Work

### ISSUE THREAD

**Issue by mtribone** [samvera-labs/nurax-pre2023#124]
### Descriptive summary

When creating a new generic work and using the WAVE accessibility toolbar, there are several form controls that have empty form labels.

They are:

1. Location (2)
1. Relationships (2)

### Rationale

A `<label>` element that is associated to a form control but does not contain text will not present any information about the form control to the user.

### Expected behavior

All form controls should have labels _with_ content.

### Actual behavior

Empty form labels.

### Steps to reproduce the behavior

Select Works from the left-hand menu
On the works screen select "Add new work"
Select "generic Work" from the modal
Validate the form with the WAVE toolbar

### Related work

#123
